### PR TITLE
chore: deploy website

### PR DESCRIPTION
**Pull request by iamogbz**
### Checklist
- [ ] This PR has updated documentation
- [ ] This PR has sufficient testing

### DevQA Prep

- Please `make node_modules`, this PR uses a new package

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -93,9 +93,10 @@ jobs:
       - save_env_cache
       - run: yarn build
       - persist_to_workspace:
-          root: ./
+          root: ../
           paths:
-            - build/
+            - docs/
+            - website/build/
```
Comment (mcataford) [inline .circleci/config.yml]: `working_directory` already sets this stage up with `website/` as its root.

Comment (iamogbz) [reply .circleci/config.yml]: Changed the `persist_to_workspace` to be the parent folder. Testing that the `docusaurus_build` relies on the docs not the build folder.

Comment (mcataford) [reply .circleci/config.yml]: 👍

Comment (iamogbz) [reply .circleci/config.yml]: Probably don't need to persist this path then, also might as well just regenerate the docs

Comment (mcataford) [reply .circleci/config.yml]: Sounds good to me

**Pull request merged**